pktoken/pktoken.go

```diff
@@ -53,12 +54,8 @@ type PKToken struct {
 
 // kid isn't always present, and is only guaranteed to be unique within a given key set,
 // so we can use the thumbprint of the key instead to identify it at verification time
-func (p *PKToken) AddJKTHeader(opKey crypto.PublicKey) error {
-	public, err := jwk.FromRaw(opKey)
-	if err != nil {
-		return fmt.Errorf("failed to create JWK from public key: %w", err)
-	}
-	thumbprint, err := public.Thumbprint(crypto.SHA256)
+func (p *PKToken) AddJKTHeader(opKey jwk.Key) error {
```

Comment:
Does this change impact downstream projects?

Reply:
The typical set of downstream projects I check are:
* All the projects in OpenPubkey org
* then the sshpiper code

In this case Docker added this specifically for something they were building, but I don't see where they use it so the code this depends on probably isn't public.

This change is seems out of scope for this PR. If there some compelling reason to make this change (Is there a bug I'm missing?), open a seperate PR and make the case in that PR so we can manage the downstream dependencies independently.

Reply:
I have changed it back [bd5d2f4](https://github.com/openpubkey/openpubkey/pull/88/commits/bd5d2f4aae0929b6db358df793330ae4982dd9dd)

However, if this is a requirement of PRs going forward, then you should add it to the agenda of either the community or TSC meeting.